model/ddl.go

```diff
@@ -241,7 +246,7 @@ func (job *Job) IsSynced() bool {
 
 // IsDone returns whether job is done.
 func (job *Job) IsDone() bool {
-	return job.State == JobStateDone
+	return job.State == JobStateDone || job.State == JobStateRollbackDone
```

Comment:
The method name `IsDone` seems not consistent with its implementation.
We can keep the original code and use `job.IsDone() || job.IsRollbackDone()` where needed.